config/sync/field.storage.eventseries.field_branch.yml

```diff
@@ -13,7 +13,7 @@ settings:
   target_type: node
 module: core
 locked: false
-cardinality: 1
+cardinality: -1
```

Comment:
Now that we have updated the cardinality then please remember to update one of the example events with multiple branches - or create a new one.